meta-mender-imx6ul/templates/imx6ulevk.conf.new

```diff
@@ -0,0 +1,23 @@
+#@TYPE: Machine
+#@NAME: NXP i.MX6UL Evaluation Kit
+#@SOC: i.MX6UL
+#@DESCRIPTION: Machine configuration for NXP i.MX6UL EVK
+#@MAINTAINER: Daniel Selvan, Jasmin Infotech <daniel.selvan@jasmin-infotech.com>
+
+MACHINEOVERRIDES =. "mx6:mx6ul:"
+
+include conf/machine/include/imx-base.inc
+include conf/machine/include/tune-cortexa7.inc
+
+MACHINE_FEATURES += " pci wifi bluetooth"
+
+IMX_DEFAULT_BOOTLOADER_mx6ul = "u-boot-imx"
+
+KERNEL_DEVICETREE = "imx6ul-14x14-evk-emmc.dtb "
+KERNEL_DEVICETREE_use-mainline-bsp = "imx6ul-14x14-evk.dtb"
+
+UBOOT_CONFIG ??= "sd"
+UBOOT_CONFIG[sd] = "mx6ul_14x14_evk_config,sdcard"
+UBOOT_CONFIG[emmc] = "mx6ul_14x14_evk_emmc_config,sdcard"
+UBOOT_CONFIG[qspi1] = "mx6ul_14x14_evk_qspi1_config"
+UBOOT_CONFIG[mfgtool] = "mx6ul_14x14_evk_config"
```

Comment:
What motivates creating a new machine type? Is there not a `imx6ulevk.conf` upstream?